kclvm/api/src/service/ty.rs

```diff
@@ -62,6 +65,9 @@ fn get_schema_ty_attributes(schema_ty: &SchemaType, line: &mut i32) -> HashMap<S
         if key != SCHEMA_SETTINGS_ATTR_NAME {
             let mut ty = kcl_ty_to_pb_ty(&attr.ty);
             ty.line = *line;
+            ty.description = attr.doc.clone().unwrap_or_default();
+            ty.filename = schema_ty.filename.clone();
+            ty.pkg_path = schema_ty.pkgpath.clone();
```

Comment:
If the type of attribute is a schema type, seems the `ty.filename` and `ty.pkg_path` converted from `kcl_ty_to_pb_ty(&attr.ty)` will be mis-overwritten?

Reply:
> If the type of attribute is a schema type, seems the `ty.filename` and `ty.pkg_path` converted from `kcl_ty_to_pb_ty(&attr.ty)` will be mis-overwritten?

Good catch! I avoided directly overwriting the package information, and the file name and pkgpath of the attributes can be taken from its parent node and add more test cases.